install/upgrade.php

```diff
@@ -58,6 +58,7 @@ public function upgrade() {
 		}
 
 		delete_transient( 'pll_languages_list' );
+		delete_option( '_transient_pll_languages_list' ); // Force deletion of the transient from the options table in case external object cache is used.
```

Comment:
Why is it necessary? I suppose that the action added above will do its job. Am I wrong?

Reply:
For the added hooks to be effective, we must delay the deletion. Done in b01def54.